Harvest.Web/ClientApp/src/Requests/ApprovalContainer.tsx

```diff
@@ -47,7 +48,7 @@ export const ApprovalContainer = () => {
     });
 
     if (response.ok) {
-      window.location.pathname = `/Project/Details/${projectId}`;
+      history.push(`/Project/Details/${projectId}`);
```

Comment:
@wongband @sprucely had a good point here -- use history.replace instead of .push since we don't want the person to hit the back button to come back to this page (though if they were to do that we should be able to handle it properly via the onload in the future)

Reply:
Okay I can change that @srkirkland should I also change it in the ExpenseContainer and QuoteContainer since they also save/submit in a form?

Reply:
I would say we don't have to bother since there is no real issue with submitting multiple requests or expenses and id like to err on the side of not replacing browsing history.

Reply:
okay sounds good